src/ripple/protocol/jss.h

```diff
@@ -139,6 +139,10 @@ JSS ( both );                       // in: Subscribe, Unsubscribe
 JSS ( both_sides );                 // in: Subscribe, Unsubscribe
 JSS ( build_path );                 // in: TransactionSign
 JSS ( build_version );              // out: NetworkOPs
+JSS ( bytes_recv_avg );             // out: Peers
```

Comment:
This (and the next) should probably be "avg_bps_recv" and "avg_bps_sent"

Reply:
to mirror that I renamed the other ones to `total_bytes_recv` and `total_bytes_sent`